models/providers/Adobe9.cfc

```diff
@@ -33,7 +36,306 @@ component accessors=true extends='cfconfig-services.models.BaseAdobe' {
 		setDotNetConfigTemplate( expandPath( '/cfconfig-services/resources/adobe9/neo-dotnet.xml' ) );
 		setVersion( '9' );
 		
+		setSeed( '0yJ!@1$r8p0L@r1$6yJ!@1rj' );
+    	setAlgorithm( 'DESede' );
```

Comment:
Again, I'm not sure why you're setting this.  These are already the default settings in the adobe password manager so you shouldn't need to set them at all.
https://github.com/bdw429s/adobe-password-util/blob/master/models/PasswordManager.cfc#L13-L14
I suppose it doesn't hurt anything, but just wanted to make that clear.